src/backendTest/kotlin/com/jakewharton/posterbox/ConfigTest.kt

```diff
@@ -99,23 +95,23 @@ class ConfigTest {
 	}
 
 	@Test fun plexHostMissingThrows() {
-		val t = assertFailsWith<KtomlException> {
+		val t = assertFailsWith<TomlDecodingException> {
 			Config.parseFromToml("""
 			|[plex]
 			|token = "abc123"
 			|""".trimMargin())
 		}
-		assertTrue("Missing the required field <host>" in t.message!!)
+		assertContains(t.message!!, "Missing required property <host>")
```

Comment:
May be `assertContains` looks better?

Reply:
Yes, thanks! I didn't know this existed.